.cloudbees/workflows/workflow.yaml

```diff
@@ -11,7 +11,66 @@ permissions:
   id-token: write
 
 jobs:
+  build-base-kaniko-image:
+    steps:
+    - name: Checkout this repo (for workflow, scripts, etc.)
+      uses: cloudbees-io/checkout@v1
+
+    - name: Checkout Kaniko upstream repo into subdir
+      uses: cloudbees-io/checkout@v1
+      with:
+        repository: chainguard-dev/kaniko
+        path: kaniko-src
+        ref: v1.25.0
+
+    - name: Login to AWS
+      uses: cloudbees-io/configure-aws-credentials@v1
+      with:
+        aws-region: us-east-1
+        role-to-assume: ${{ vars.oidc_staging_iam_role }}
+        role-duration-seconds: "3600"
+
+    - name: Configure container registry for ECR
+      uses: cloudbees-io/configure-ecr-credentials@v1
+
+    - name: Append GOFLAGS and patch Dockerfile
+      uses: docker://golang:1.24.0
+      run: |
+        cd kaniko-src
+        git config user.name "ci"
+        git config user.email "ci@example.com"
+
+        # Insert GOFLAGS after ARG TARGETOS
+        sed -i '/^ARG TARGETOS/a ARG GOFLAGS="-buildvcs=false"' deploy/Dockerfile
```

Comment:
I thought we agreed you'd create a git diff of the tag and use `git apply` to apply it?